### PR TITLE
Update step.yml

### PR DESCRIPTION
**Pull request by johnsenner**
Fixed a spelling mistake and added some default fields!

### CODE REVIEW

**Comment by johnsenner** [comment]
Apologies for the check failure - I'm new to yaml and I don't have an env set up to test it. And there's probably a lot I'm not even aware of! Just wanted to add some helpful default stuff like the slack chat plugin has. 😃

File: step.yml [hunk 1]
```diff
@@ -61,19 +61,24 @@ inputs:
       title: "Send To emails"
       description: |
         Comma separated email address list.
-      summary: ""
+      summary: "
```
Comment (viktorbenei) [inline step.yml]: for empty string please use `""` - or just remove the `summary` item, it's an empty string by default ;)

File: step.yml [hunk 2]
```diff
@@ -61,19 +61,24 @@ inputs:
...
       description: ""
       summary: ""
       is_required: true
-  - message: "Build succed on Bitrise.io"
+  - message: "$BITRISE_APP_TITLE Build $BITRISE_BUILD_NUMBER succeeded on Bitrise.io"
     opts:
       title: "Email Message"
       description: |
-        The message can be HTML or plain text.
+        Build succeeded!
```
Comment (viktorbenei) [inline step.yml]: please don't change the description - it's not the default value of the step but the description of the input, which is presented on the UI ;)

**Comment by viktorbenei** [comment]
No problem @johnsenner , thanks for the PR! ;)

To test this step on your own Mac/Linux:

1. git clone it
1. and use our CLI to run test with it; specify these as secrets: https://github.com/bitrise-io/steps-email-with-mailgun/blob/master/bitrise.yml#L8 then you can run `bitrise run test` to perform the tests

One more thing, you can even check the step in the Workflow Editor, as you can run the editor right from the CLI! Related guide: https://discuss.bitrise.io/t/how-to-experiment-with-bitrise-configs-locally-on-your-mac-linux/1751

Note 2: you can also try your own version anywhere including bitrise.io, by using a git clone URL pointing to your fork! ;) Related docs: https://devcenter.bitrise.io/bitrise-cli/steps/#step-referenceid-format (check the `git::` format).

If you'd have any questions just let us know! ;)

**Comment by johnsenner** [comment]
Totally, good call

**Comment by johnsenner** [comment]
Updated `error_message`

**Comment by viktorbenei** [comment]
Available in the latest version, thanks again @johnsenner for the PR! 🚀

**Pull request merged**